api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessConstants.java

```diff
@@ -20,4 +20,11 @@
     public static final String PERSONATTRIBUTETYPE_FORMAT = "java.lang.String";
     public static final String PERSONATTRIBUTETYPE_UUID = "0a93cbc6-5d65-4886-8091-47a25d3df944";
 
+    public static final String PATIENT_RESTRICTION_GLOBAL_PROPERTY_NAME = "locationbasedacess.patientaccess";
```

Comment:
what do you think about - `locationbasedaccess.access.patient`? like this for others?

Reply:
fixed it. please check now